src/runtime/virtcontainers/clh.go

```diff
@@ -1405,17 +1405,16 @@ func (clh *cloudHypervisor) isClhRunning(timeout uint) (bool, error) {
 		return false, nil
 	}
 
-	err := syscall.Kill(pid, syscall.Signal(0))
-	if err != nil {
-		return false, nil
-	}
-
 	timeStart := time.Now()
 	cl := clh.client()
 	for {
```

Comment:
I think this commit message (`clh: prevent 10 seconds timeout when checking for clh`) also needs to be refined to explain the purpose of the patch.

To my understanding:

```
clh: Return faster with dead Cloud Hypervisor process from `isClhRunning()`

Through more proactively checking if Cloud Hypervisor process is dead, this patch
provides a faster path for `isClhRunning()`.
```

Reply:
You're right, I changed the commit message.